src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -136,15 +132,16 @@ public void onAuthConfirmCall(@RequestBody LinkOnConfirmResponse linkOnConfirmRe
   @PostMapping({"/v0.5/links/link/on-add-contexts"})
   public void onAddCareContext(
       @RequestBody LinkOnAddCareContextsResponse linkOnAddCareContextsResponse) {
-    log.info(linkOnAddCareContextsResponse.toString());
+    log.debug(linkOnAddCareContextsResponse.toString());
     if (linkOnAddCareContextsResponse != null) {
-      if (linkOnAddCareContextsResponse.getError() != null)
+      if (linkOnAddCareContextsResponse.getError() != null) {
         log.error(
             "/v0.5/links/link/on-add-contexts error: "
                 + linkOnAddCareContextsResponse.getError().getMessage());
+      }
       requestLogService.setHipOnAddCareContextResponse(linkOnAddCareContextsResponse);
     } else {
-      log.info("Failed to add Context");
+      log.error("Failed to add Context");
```

Comment:
In case of invalid incoming data, we need to respond back to the caller saying it is an Invalid request, we can refer to the API spec in the ABDM for the response structure in case of error.

Reply:
The caller here is Gateway. I am not sure sending error messages to Gateway will be of value-add.

Reply:
Created https://github.com/NHA-ABDM/ABDM-wrapper/issues/37 to address this